src/uu/rmdir/src/rmdir.rs

```diff
@@ -16,6 +16,7 @@ use std::io;
 use std::path::Path;
 use uucore::display::Quotable;
 use uucore::error::{set_exit_code, strip_errno, UResult};
+use uucore::fs::is_symlink;
```

Comment:
clippy isn't happy here; I guess it is confused by the implementation of is_symlink in this file